### DISCUSSION TITLE
Documenting platform (e.g. `arm64`, `amd64`) for `ubuntu-24.04`

### DISCUSSION THREAD

**Discussion by jamesbraza** [actions/virtual-environments#10824] [General questions]
I can't find it [here](https://github.com/actions/runner-images/blob/win22/20241015.1/images/ubuntu/Ubuntu2404-Readme.md) or [here](https://github.com/actions/runner-images/tree/win22/20241015.1?tab=readme-ov-file#available-images), what is the platform for `ubuntu-24.04`?

Can we add this information to one of those pages?

**Comment by Sainan** [comment]
Pretty sure it's amd64 unless otherwise specified.

**Comment by Stromweld** [comment]
with the announcement of ARM versions of ubuntu and windows can we get them added to the README table for this repo so it's easier to identify the tags and have more information on what's available? https://github.blog/changelog/2024-09-03-github-actions-arm64-linux-and-windows-runners-are-now-generally-available/
